rspec-rails.gemspec

```diff
@@ -27,9 +27,9 @@ Gem::Specification.new do |s|
     s.cert_chain = [File.expand_path('~/.gem/rspec-gem-public_cert.pem')]
   end
 
-  s.add_runtime_dependency(%q<activesupport>, [">= 3.0"])
-  s.add_runtime_dependency(%q<actionpack>, [">= 3.0"])
-  s.add_runtime_dependency(%q<railties>, [">= 3.0"])
+  s.add_runtime_dependency(%q<activesupport>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<actionpack>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<railties>, [">= 3.0", "<= 4.2"])
```

Comment:
If say a 4.3 does come out and we don't need code changes, I see no problems in releasing patch dot release bumping the gem support.